### ISSUE TITLE
Scripting console: API (?) to automate clearing the console

### ISSUE THREAD

**Issue by Tom-Willemsen** [ISISComputingGroup/IBEX#5843]
The output of the scripting console in the GUI has a limited size to avoid it taking large amounts of memory. As a muon scientist I would like to automatically clear the console at certain times, so that I don't run into this limit mid-experiment.

The suggested implementation of this ticket is to have an API in genie_python that talks to the GUI and tells it to clear the console. This is done elsewhere using py4j. The scientists would then hook into this API to clear the console automatically when, for example, a new RB number is set.

# Acceptance criteria:
- Console can be cleared in an automated way

**Comment by ChrisM-S** [comment]
Is it possible to do this incrementally once it reaches a buffer size limit (following the FIFO pattern as with a windows terminal)?  It does make sense to have an equivalent of the "cls" command as hook to clear everything though.  It might be irritating if the automated clear of the buffer was an arbitrary "cls".

**Comment by Tom-Willemsen** [comment]
@ChrisM-S yes that would be ideal but is hard in pydev. See discussion on https://github.com/ISISComputingGroup/IBEX/issues/4146

**Comment by John-Holt-Tessella** [comment]
What problem are they actually trying to solve having the console open for a long time. They should be able to restart the GUI at any point so what information are they trying to keep? Should we be saving it somewhere more permanent and associating it with a run?
